examples/Console/TestGrpcNetClient.cs

```diff
@@ -0,0 +1,57 @@
+﻿// <copyright file="TestGrpcNetClient.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using Examples.GrpcService;
+using Grpc.Core;
+using Grpc.Net.Client;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+
+namespace Examples.Console
+{
+    internal class TestGrpcNetClient
+    {
+        internal static object Run()
+        {
+            using var openTelemetry = Sdk.CreateTracerProviderBuilder()
+                .AddGrpcClientInstrumentation()
```

Comment:
it could be obvious, but would be great if we mention somewhere that the GrpcClient instrumentation is only suported in .net core 3.1 or above!

Reply:
I could update the readme for the instrumentation with this information. Perhaps also just add a comment in the code of this example as well.